mavis/test/pages/sessions.py

```diff
@@ -828,3 +849,14 @@ def check_child_has_psd(self, child: Child) -> None:
             f'div.nhsuk-card.app-card.app-card--compact:has(h4:has-text("{child!s}"))'
         )
         expect(patient_card).to_contain_text("PSD added")
+
+    @step("Click on Send reminders")
+    def click_send_reminders(self, school: School) -> None:
+        self.send_reminders_link.click()
+        self.expect_text_to_not_be_visible("Sorry, there is a problem with the service")
+        expect(
+            self.page.get_by_role("heading", name="Manage consent reminders")
+        ).to_be_visible()
+        self.page.get_by_role(
+            "link", name=school.name
+        ).click()  # Update when MAV-2048 is done
```

Comment:
Worth checking with the designers about MAV-2048, usually a back link is not used if there are breadcrumbs.